src/shell/lisa_shell

```diff
@@ -318,14 +321,22 @@ EOF
 }
 
 function _lisa-test {
-nosetests -v --nocapture --nologcapture \
-          --logging-config=logging.conf \
-          $*
+if [ -n "$LISA_RESULTS_DIR" ]; then
+  mkdir -p "$LISA_RESULTS_DIR"
+  # There is no sensible place to store results.xml when the directory is not
+  # set explicitely, as the Python scripts decide where to put the results..
+  # Using results_latest symlink is not an option as it would race if using the
+  # same tree to run multiple tests on different boards at the same time.
+  local xunit_options=(--with-xunit --xunit-file="$LISA_RESULTS_DIR/results.xml")
```

Comment:
Ah, that explains the environment variable. We could read the `results_dir` option from the file conf but that means reading JSON in bash...